integration/exec_test.go

```diff
@@ -12,6 +12,17 @@ import (
 )
 
 func TestExecPS(t *testing.T) {
+	testExecPS(t, false)
+}
+
+func TestUsernsExecPS(t *testing.T) {
+	if _, err := os.Stat("/proc/self/ns/user"); os.IsNotExist(err) {
```

Comment:
@LK4D4 Most likely because we assume that the other namespaces are available whereas NEWUSER was only enabled over the last year in most distributions. RHEL 7 did not have it enabled for e.g. but 7.1 will. I agree that testing should be better integrated with the kernel config.